From 176d588c5e080b4acc97c8ce1a81fed7121b4ba8 Mon Sep 17 00:00:00 2001
From: Kumar Gala <galak@kernel.crashing.org>
Date: Tue, 9 Mar 2010 14:13:34 -0600
Subject: [PATCH 115/233] fsl_pq_mdio: Add fman to compatiable list for the driver

Extracted from vendor drop QorIQ-DPAA-SDK-V1-20110609-systembuilder.iso
0145-fsl_pq_mdio-Add-fman-to-compatiable-list-for-the-dri.patch

Signed-off-by: Kumar Gala <kumar.gala@freescale.com>
---
 drivers/net/fsl_pq_mdio.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/net/fsl_pq_mdio.c b/drivers/net/fsl_pq_mdio.c
index 50c5f00..3d489ad 100644
--- a/drivers/net/fsl_pq_mdio.c
+++ b/drivers/net/fsl_pq_mdio.c
@@ -324,6 +324,7 @@ static int fsl_pq_mdio_probe(struct of_device *ofdev,
 
 	if (of_device_is_compatible(np, "fsl,gianfar-mdio") ||
 			of_device_is_compatible(np, "fsl,gianfar-tbi") ||
+			of_device_is_compatible(np, "fsl,fman-mdio") ||
 			of_device_is_compatible(np, "fsl,ucc-mdio") ||
 			of_device_is_compatible(np, "ucc_geth_phy"))
 		map -= offsetof(struct fsl_pq_mdio, miimcfg);
-- 
1.7.0.4

